base.py

```diff
@@ -130,12 +130,14 @@ def log_encounter(human1, human2, location, duration, distance, time):
                             'is_infected': human1.is_exposed or human1.is_infectious,
                             'infectiousness': human1.infectiousness,
                             'symptoms': human1.symptoms,
+                            'preexisting_conditions': human1.preexisting_conditions,
```

Comment:
It might be observed from the perspective of human 1. We should confirm it tomorrow at the meeting. Might be human2's preexisting conditions are unobserved for human1 due to privacy issues.

Reply:
I think it's unobserved for now; I left it like this.